src/Pester.Runtime.ps1

```diff
@@ -1216,7 +1227,7 @@ function Invoke-PluginStep {
     # this is end step, we should run all steps no matter if some failed, and we should run them in opposite direction
     # only do this if there is more than 1, to avoid the "expensive" -like check and reverse
     $isEndStep = 1 -lt $pluginsWithGivenStep.Count -and $Step -like "*End"
-    if (-not $isEndStep) {
+    if ($isEndStep) {
```

Comment:
So this did not reverse the steps before, did you see any bugs linked to that?

Reply:
No bugs I'm aware of. Just not behaving like comment says. We reverse on TearDown-steps as expected, but this currently reverses on everything else than *End.

Not a critical part of the PR, just testing it out while in draft.